ec2/spark_ec2.py

```diff
@@ -643,12 +654,14 @@ def setup_cluster(conn, master_nodes, slave_nodes, opts, deploy_ssh_key):
 
     # NOTE: We should clone the repository before running deploy_files to
     # prevent ec2-variables.sh from being overwritten
+    repo_branch="{r} -b {b}".format(r=opts.spark_ec2_git_repo, b=opts.spark_ec2_branch)
+    print "Cloning spark-ec2 scripts from {rb} on master....".format(rb=repo_branch)
```

Comment:
This print looks incorrect.

> Cloning spark-ec2 scripts from https://github.com/mesos/spark-ec2 -b branch-1.3 on master....

Should probably instead be something like:

> Cloning spark-ec2 scripts from https://github.com/mesos/spark-ec2/tree/branch-1.3 ...

Also, formatting nit: Typically we use 3 dots `...` to end statements like these.